tests/foreman/cli/test_setting.py

```diff
@@ -265,32 +270,41 @@ def test_negative_update_email_subject_prefix(self):
 
     @stubbed()
     @tier1
-    def test_positive_update_send_welcome_email(self):
+    def test_negative_update_send_welcome_email(self):
         """Check email send welcome email is updated
 
-        :id: cdaf6cd0-5eea-4252-87c5-f9ec3ba79ac1
+        :id: 2f75775d-72a1-4b2f-86c2-98c36e446099
 
-        :steps: valid values: boolean true or false
+        :steps: set invalid values: not booleans
 
-        :expectedresults: send_welcome_email is updated
+        :expectedresults: send_welcome_email is not updated
 
         :caseautomation: notautomated
 
         :caseimportance: low
         """
 
-    @stubbed()
-    @tier1
-    def test_negative_update_send_welcome_email(self):
-        """Check email send welcome email is updated
 
-        :id: 2f75775d-72a1-4b2f-86c2-98c36e446099
+@tier1
+@pytest.mark.parametrize('value', [u'true', u'false'])
+def test_positive_update_send_welcome_email(value):
+    """Check email send welcome email is updated
 
-        :steps: set invalid values: not booleans
+    :id: cdaf6cd0-5eea-4252-87c5-f9ec3ba79ac1
 
-        :expectedresults: send_welcome_email is not updated
+    :steps: valid values: boolean true or false
 
-        :caseautomation: notautomated
+    :expectedresults: send_welcome_email is updated
 
-        :caseimportance: low
-        """
+    :caseautomation: automated
+
+    :caseimportance: low
+    """
+
+    SettingsSetCmd().name_option(u'send_welcome_email').value(value).execute()
+    settings = {
+        setting[u'name']: setting[u'value']
+        for setting in SettingsListCmd().execute()
+    }
+
+    assert value == settings[u'send_welcome_email']
```

Comment:
Just curious, in test results, will this come up as one test with two results, or two separate tests?

Reply:
On execution each test is different. In can pass ids if want be more clear about the differences. Otherwise the parameters will be used for each test case. In fact I'm curious to check xml report to check what will be there and how data will be feed into Polarion ;)